src/pngwriter.cc

```diff
@@ -2023,6 +2023,11 @@ void pngwriter::write_png(void)
 ///////////////////////////////////////////
 void pngwriter::plot_text( char * face_path, int fontsize, int x_start, int y_start, double angle, char * text, double red, double green, double blue)
```

Comment:
I am not sure a non-const `char *` is needed at all. can you just transform the old one?